packages/react-ai/src/components/AIConversation/types.ts

```diff
@@ -54,6 +55,11 @@ export interface AIConversation {
 
 export type MessageVariant = 'bubble' | 'default';
 
+export interface MessageRenderer {
+  text?: (message: string) => React.JSX.Element;
```

Comment:
Would it make sense to expand the function definition here to allow an object so additional key/values can be included in the future?
```
  // example
  text?: (text: { content: string )) => React.JSX.Element;
```